libs/ledger-live-common/src/families/tron/types.ts

```diff
@@ -101,18 +118,36 @@ export type SmartContractTransactionData = {
   parameter: string;
   owner_address: string;
 };
-export type UnfreezeTransactionData = {
-  receiver_address?: string;
+
+export type FreezeTransactionData = {
   owner_address: string;
+  frozen_balance: number;
   resource: TronResource | null | undefined;
 };
-export type FreezeTransactionData = {
+
+export type UnFreezeTransactionData = {
+  owner_address: string;
+  resource: TronResource | null | undefined;
+  unfreeze_balance: number;
+};
+
+export type LegacyUnfreezeTransactionData = {
```

Comment:
It's essentially the same as before. It might appear differently due to how Git is visualizing the changes.

Before:
```
export type UnfreezeTransactionData = {
  receiver_address?: string;
  owner_address: string;
  resource: TronResource | null | undefined;
};
```

After:
```
export type LegacyUnfreezeTransactionData = {
  receiver_address?: string;
  owner_address: string;
  resource: TronResource | null | undefined;
};
```